microceph/cmd/microceph/disk_list.go

```diff
@@ -71,34 +78,66 @@ func (c *cmdDiskList) Run(cmd *cobra.Command, args []string) error {
 }
 
 func listLocalDisks(cli *microCli.Client) error {
+	data := [][]string{}
+
+	disks := get_unpartitioned_disks(cli)
+	for _, disk := range disks {
+		data = append(data, []string{disk.Model, disk.Size, disk.Type, disk.Path})
+	}
+
+	fmt.Println("")
+	fmt.Println("Available unpartitioned disks on this system:")
+
+	header := []string{"MODEL", "CAPACITY", "TYPE", "PATH"}
+	sort.Sort(lxdCmd.SortColumnsNaturally(data))
+
+	err := lxdCmd.RenderTable(lxdCmd.TableFormatTable, header, data, data)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func get_unpartitioned_disks(cli *microCli.Client) []Disk {
```

Comment:
I'm in favor of using snake case, but let's stick to what Go does and use cameCase instead, for consistency's sake.

Reply:
Also, why not return a tuple of error and Disk array? This function seems to have a different convention for no reason.

Reply:
sorry I've been writing some python code on the side and being haunted by snakes and camels right now. Will make modifications accordingly.